sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -6577,9 +6584,13 @@ pi_result piEnqueueEventsWaitWithBarrier(pi_queue Queue,
     if (auto Res = Queue->executeCommandList(CmdList, false, OkToBatch))
       return Res;
 
-  // We must keep the event internally to use if new command lists are created.
-  (*Event)->RefCount.increment();
-  Queue->ActiveBarriers.push_back(*Event);
+  if (*Event) {
```

Comment:
why could *Event be null here now?

Reply:
If queues are unused, there will be no Event created.